WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1270,7 +1270,7 @@ ConvView                    = @PanelBorder@  @Conv2Bg@; box-sizing: border-box;
 Conv2Bg                     = @AppBg@
 Conv2View                   = height: 100%;  margin: @MailViewGutterSpace@;
 Conv2Header                 = background-color: transparent; overflow: hidden; @ActiveCursor@; width: 100%; max-width100%; padding:@MailViewGutterSpace@ 0; margin-bottom: @MailViewInnerSpacing@;
-Conv2HeaderSubject          = @FontSize-biggest@ ;padding: 0 5px; width:95%;@Text@
+Conv2HeaderSubject          = @FontSize-biggest@ ;padding: 0 5px; width:95%; @Text@; @NoWrap@;
```

Comment:
ok if it turns out to be too complex then we can leave it as that is current behaviour